src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -122,16 +122,22 @@ const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
       ...((bordered || v.borderColor || circular) &&
         getBorderedStyles(circular, v.borderColor || getIconColor(color, v))),
 
-      ...(rtl && {
-        transform: `scaleX(-1) rotate(${-1 * rotate}deg)`,
-      }),
-
       ...(!rtl && {
         transform: `rotate(${rotate}deg)`,
       }),
     }
   },
 
+  rotateInRtl: ({ props: p, theme }) => {
```

Comment:
```suggestion
  rotateInRtl: ({ props: p, theme: { rtl } }) => ({
    ...(rtl && {
      transform: `scaleX(-1) rotate(${-1 * p.rotate}deg)`,
    }),    
  })
```

We can make this more compact 🤔